gomod-check/action.yml

```diff
@@ -0,0 +1,28 @@
+name: "go.mod check"
+description: "Checks that the go.mod doesn't contain replacements for 'protected' modules. Assumes go, jq and grep are available."
```

Comment:
Out of curiosity - what do we need the `jq` for? I don't see it to be used down in the script...